src/comp/Loading.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from "react"
+import "./Loading.css"
+
+class Loading extends React.Component {
+	render() {
+		return (
+			<div className="loader" />
+		)
+	}
+}
```

Comment:
Short components (like this) can be written as function component to reduce lines.
```suggestion
function Loading():JSX.Element {
	return <div className="loader" />
}
```